tests/test_torch_compile.py

```
@@ -0,0 +1,202 @@
+    model.on_train_begin()
+    model.init_optimizers()
+
+    batch_size = 1
+    labels = torch.nn.functional.one_hot(torch.randint(0, 10, (batch_size,)), num_classes=10).float()
+    data = {
+        "real": torch.randn(batch_size, 3, 8, 8).to(model.device, model.precision),
+        "condition": labels.to(model.device, model.precision),
+        "neg_condition": torch.zeros(batch_size, 10).to(model.device, model.precision),
+    }
+
+    loss_map, outputs = model.single_train_step(data, 0)
```

Ruff (0.15.15)

[warning] Unpacked variable `outputs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)